match/lib/match/options.rb

```diff
@@ -80,6 +80,11 @@ def self.available_options
                                      description: "Renew the provisioning profiles every time you run match",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :skip_confirmation,
+                                     env_name: "MATCH_SKIP_CONFIRMATION",
+                                     description: "Disables confirmation dialogs during nuke, answering them with yes",
```

Comment:
Sorry to keep bothering you with wording changes, but lets use "prompts" rather than "dialogs" here. So:
```Disables confirmation prompts during nuke, answering them with yes```